src/docs/patterns/forms.mdx

```diff
@@ -218,14 +214,11 @@ For additional instructions, refer to [Material Design's design specs on "Text f
 If certain dimensions are not specified, refer to the dimensions suggested in the mobile section above.
 
 <ImageGrid images={[Specs4Image, Specs5Image]} regularWidth />
```

Comment:
the 40/32 margin is redundant here, since it is the same as the mobile dimension